app/views/parties/new.html.erb

```diff
@@ -0,0 +1,74 @@
+<h3> Start a party </h3>
+
+<%= form_with url: "/parties", method: :post, data: { turbo: false } do |f| %>
+  Services </br>
+  <%= check_box_tag "services []", 337 %>
+  Disney Plus </br>
+  <%= check_box_tag "services []", 1899 %>
+  Max </br>
+  <%= check_box_tag "services []", 9 %>
+  Amazon Prime Video </br>
+  <%= check_box_tag "services []", 15 %>
+  Hulu </br>
+  <%= check_box_tag "services []", 8 %>
+  Netflix </br>
+  <%= check_box_tag "services []", 350 %>
+  Apple TV Plus </br>
+  Genres </br>
+  <%= check_box_tag "genres []", 28 %>
+  Action </br>
+  <%= check_box_tag "genres []", 12 %>
+  Adventure </br>
+  <%= check_box_tag "genres []", 16 %>
+  Animation </br>
+  <%= check_box_tag "genres []", 35 %>
+  Comedy </br>
+  <%= check_box_tag "genres []", 80 %>
+  Crime </br>
+  <%= check_box_tag "genres []", 99 %>
+  Documentary </br>
+  <%= check_box_tag "genres []", 18 %>
+  Drama </br>
+  <%= check_box_tag "genres []", 10751 %>
+  Family </br>
+  <%= check_box_tag "genres []", 14 %>
+  Fantasy </br>
+  <%= check_box_tag "genres []", 36 %>
+  History </br>
+  <%= check_box_tag "genres []", 27 %>
+  Horror </br>
+  <%= check_box_tag "genres []", 10402 %>
+  Music </br>
+  <%= check_box_tag "genres []", 9648 %>
+  Mystery </br>
+  <%= check_box_tag "genres []", 10749 %>
+  Romance </br>
+  <%= check_box_tag "genres []", 878 %>
+  Science Fiction </br>
+  <%= check_box_tag "genres []", 107770 %>
+  TV Movies </br>
+  <%= check_box_tag "genres []", 53 %>
+  Thriller </br>
+  <%= check_box_tag "genres []", 10752 %>
+  War </br>
+  <%= check_box_tag "genres []", 37 %>
+  Western </br>
+  
+  <%= f.label :max_duration %>
+  <%= f.number_field :max_duration, value: 300 %></br>
+
+  Rating:
```

Comment:
Do we want to deal with defaults? Are we storing a user's preferences right now? I know it was an idea but can't remember if we made it a stretch goal or MVP